Formula/caffe.rb

```diff
@@ -0,0 +1,53 @@
+class Caffe < Formula
+  desc "Fast open framework for deep learning"
+  homepage "http://caffe.berkeleyvision.org/"
+  url "https://github.com/BVLC/caffe/archive/1.0.tar.gz"
+  sha256 "71d3c9eb8a183150f965a465824d01fe82826c22505f7aa314f700ace03fa77f"
+
+  head "https://github.com/BVLC/caffe.git"
+
+  option "with-test", "Run tests during build step"
```

Comment:
We only allow `make check` and `make test` for security software.

Reply:
I tried to follow the suggestions supplied [in the comment here](https://github.com/Homebrew/homebrew-science/pull/2402/files#r32586505), as it seems to be not trivial to. I will remove it.

Reply:
homebrew/science is deprecated and had very different standards and procedures from homebrew/core